modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -108,26 +108,70 @@ void remapYolo(const std::unordered_map<std::string, cv::Mat> &onnx,
     // Configured output
     cv::Mat& out = gapi.begin()->second;
     // Simple copy
-    copyToOut(in, out);
+    copyToOut<float>(in, out);
+}
+
+void remapYoloV3(const std::unordered_map<std::string, cv::Mat> &onnx,
+                            std::unordered_map<std::string, cv::Mat> &gapi) {
+    // Simple copy for outputs
+    const cv::Mat& in_boxes = onnx.at("yolonms_layer_1/ExpandDims_1:0");
+    const cv::Mat& in_scores = onnx.at("yolonms_layer_1/ExpandDims_3:0");
+    const cv::Mat& in_indices = onnx.at("yolonms_layer_1/concat_2:0");
+
+    cv::Mat& out_boxes = gapi.at("out1");
+    cv::Mat& out_scores = gapi.at("out2");
+    cv::Mat& out_indices = gapi.at("out3");
+
+    copyToOut<float>(in_boxes, out_boxes);
+    copyToOut<float>(in_scores, out_scores);
+    copyToOut<int32_t>(in_indices, out_indices);
+}
+
+void remapToIESsdOut(const cv::Mat &num_detections,
+                     const cv::Mat &detection_boxes,
+                     const cv::Mat &detection_scores,
+                     const cv::Mat &detection_classes,
+                           cv::Mat &ssd_output) {
+    GAPI_Assert(num_detections.depth() == CV_32F);
+    GAPI_Assert(detection_boxes.depth() == CV_32F);
+    GAPI_Assert(detection_scores.depth() == CV_32F);
+    GAPI_Assert(detection_classes.depth() == CV_32F);
+
+    const int num_objects = static_cast<int>(num_detections.ptr<float>()[0]);
+    const float *in_boxes = detection_boxes.ptr<float>();
+    const float *in_scores = detection_scores.ptr<float>();
+    const float *in_classes = detection_classes.ptr<float>();
```

Comment:
Yes, empty or not. `num_objects` only.

Reply:
Fixed.

Reply:
But what if there are less elements than the cycle is going to consume? Why are you sure at this point about the sizes of the matrices?

Reply:
`GAPI_Assert(num_objects <= (ssd_output.size[2] - 1));`

Reply:
I'm sorry, I should've written more precise comments to avoid misunderstanding.
What I've wanted to say - the function has several input matrices (4, if exactly). But you're checking only the number given by `detections[0]` if objects can be stored in `ssd_output`.
But how can you be sure you won't call the indices of `in_boxes, in_scores, in_classes` out of their range? I suppose, they have to be bound with `num_objects` outside of this function, but here it is not checked.

Reply:
This is based on the properties of the network.

Reply:
https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/ssd-mobilenetv1#postprocessing

Reply:
So, check the sizes related to `num_objects`, please